gef-ui/server/controllers/unissued-facilities/index.test.js

```diff
@@ -209,7 +211,7 @@ describe('postChangeUnissuedFacility()', () => {
         coverEndDate: format(tomorrow, 'MMMM d, yyyy'),
         coverStartDate: format(tomorrow, 'MMMM d, yyyy'),
         issueDate: format(now, 'MMMM d, yyyy'),
-        shouldCoverStartOnSubmission: null,
+        shouldCoverStartOnSubmission: false,
```

Comment:
should this be a string? i.e. `false`
I've seen in the validation file that you are using something like `shouldCoverStartOnSubmission === 'false'`

Reply:
ah. never mind. that's the payload that we send to the server

Reply:
ah. never mind. that's the payload that we send to the server